File: Source/Rio.Web/src/app/shared/models/posting/posting-dto.ts
import { PostingTypeDto } from './posting-type-dto';
import { PostingStatusDto } from './posting-status-dto';
import { AccountDto } from '../account/account-dto';

export class PostingDto {
    PostingID: number;
    PostingType: PostingTypeDto;
    PostingStatus: PostingStatusDto;

    PostingDescription: string;
    Quantity: number;
    AvailableQuantity: number;
    Price: number;

    PostingDate: Date;

    CreateAccount: AccountDto;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Source/Rio.Models/DataTransferObjects/Posting/PostingDto.cs
using Rio.Models.DataTransferObjects.Account;
using System;

namespace Rio.Models.DataTransferObjects.Posting
{
    public class PostingDto
    {
        public int PostingID { get; set; }
        public PostingTypeDto PostingType { get; set; }
        public DateTime PostingDate { get; set; }
        public int Quantity { get; set; }
        public int AvailableQuantity { get; set; }
        public decimal Price { get; set; }
        public string PostingDescription { get; set; }

        public AccountDto CreateAccount { get; set; }
        public PostingStatusDto PostingStatus { get; set; }
    }
}
